adazza-scala.gradle

```diff
@@ -5,9 +5,27 @@
  desired project scope explicitly.
 */
 
+
 apply plugin: 'scala'
+
+if (!project.hasProperty("scalaVersion") || !['2.11', '2.12'].contains(scalaVersion)) {
```

Comment:
One half of this is a default the other half is an error. If I don't pass in anything we want scala 12, if I pass in garbage we want to fai.